app/src/main/res/layout/fragment_menu.xml

```diff
@@ -1,42 +1,165 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:ignore="ContentDescription"
-    tools:context=".presentation.menu.MenuFragment">
+    android:layout_marginTop="@dimen/margin_top_toolbar"
+    android:background="@drawable/bg_menu"
+    tools:context=".presentation.menu.MenuFragment"
+    tools:ignore="ContentDescription">
```

Comment:
Use `ignore` only in specific views, otherwise you might miss important lint warnings